File: cartoon-admin/src/app/pages/auth/auth.service.ts
import { Injectable } from '@angular/core';
import {BaseService} from "../../services/base.service";
import {AuthRequest} from "./auth.model";

@Injectable({
  providedIn: 'root'
})
export class AuthService{
  private readonly apiUrl: string = "/api/v1/auth"
  constructor(private base : BaseService) {
  }
  login(request:AuthRequest){
    return this.base.post(this.apiUrl+"/login",request);
  }

}

File: business/src/main/java/com/thd/cartoon/service/AuthService.java
package com.thd.cartoon.service;

import com.thd.cartoon.dto.auth.AuthRequest;
import com.thd.cartoon.dto.response.BaseResponse;

/**
 * @author DatNuclear 16/01/2024
 * @project cartoon-movie
 */
public interface AuthService {
    BaseResponse login(AuthRequest request);
}
